subsys/bluetooth/host/hci_core.c

```diff
@@ -7401,6 +7411,15 @@ int bt_le_per_adv_sync_create(const struct bt_le_per_adv_sync_param *param,
 		cp->cte_type |= BT_HCI_LE_PER_ADV_CREATE_SYNC_CTE_TYPE_ONLY_CTE;
 	}
 
+	if (param->options &
+		BT_LE_PER_ADV_SYNC_OPT_REPORTING_INITIALLY_DISABLED) {
```

Comment:
Got some really long names here. :)

Should be align within the if-statement.

Reply:
Any suggestions for shorter names are welcome :D 

Regarding the alignment, I don't think the style guide mandates anything, and it seems to be a mix of what is being used in Zephyr. 

Personally I prefer this over
```
	if (param->options &
	    BT_LE_PER_ADV_SYNC_OPT_REPORTING_INITIALLY_DISABLED) {
```
as it makes it more clear that the two are connected, and gives some difference between long lines that are split, and multiple expressions (e.g. multiple `&&` or `||`).

If you still want them to be aligned, please let me know, but it does seem like additional indentation is the most commonly used in Zephyr.

Reply:
Sorry, no good suggestions, it's fine though. Just hard with the 80 limit.

Atleast for the bluetooth code it is always aligned on the `(`, so I would prefer that.

Reply:
Done :)